distribution/client/src/lib/storage.js

```diff
@@ -19,10 +19,10 @@ class Storage {
     /* The default home directory is /evergreen, see the Dockerfile in the root
      * directory of th repository
      */
-    if (!process.env.EVERGREEN_HOME) {
-      return '/evergreen';
+    if (!process.env.EVERGREEN_DATA) {
```

Comment:
Not sure about this one. It's going to be confusing in the code. Maybe better to introduce a new utility function here or so? `jenkinsHome()` returning data, not home basically?

Reply:
`jenkinsHome()` already exist one function below this :smile: